# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/smartypants/hotfix-import.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- smartypants.py-2.0.2/setup.py.vanilla       2026-09-12 19:01:48.237565413 +0300
+++ smartypants.py-2.0.2/setup.py       2026-09-12 19:02:51.754308754 +0300
@@ -6,7 +6,7 @@
 
 import codecs
 import sys
-from distutils.core import Command, setup
+from setuptools import Command, setup
 from unittest import TestLoader, TextTestRunner
 
 try:

